types/git-username/git-username-tests.ts

```diff
@@ -2,4 +2,5 @@ import username = require('git-username');
 
 username('some/path'); // $ExpectType string | null
 username({ cwd: 'some/path' }); // $ExpectType string | null
-username({ strict: true }); // $ExpectType string
+username({ strict: true }); // $ExpectType string | null
```

Comment:
This test is not correct, when `strict` is set to `true` the function never returns `null`:
```suggestion
username({ strict: true }); // $ExpectType string
```